qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature

```diff
@@ -0,0 +1,126 @@
+	And user choses a file in <Format> format 
+	And with <Extension> extension 
+	And attached file size is less than or equal to 10 Mb
+	And attached file size is more than 0 bytes
+	Then attached file name should be displayed under the "Attach CV" button
+	Examples:
+		|Format|Extension|
+		|PDF   |.pdf     |
+		|DOC   |.doc     |
+		|DOCX  |.docx    |
+		|RTF   |.rtf     |
+	
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_04 Fill the field "Name" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Name" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Name" field
+	Examples:
+		|sample data|comment              |
+		|           |empty field          |
+		|           |one or more spaces   |
+		|Olga       |one word             |
+		|Ольга      |one word, non-English|
+		|11111553   |digits, one word     |
+		|132342 678 |digits, two words    |
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_05 Fill the field "Email" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Email" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Email" field
+	Examples:
+		|sample data |comment                  |
+		|            |empty field              |
+		|            |one or more spaces       |
+		|@.com       |empty username and domain|
+		|вася@mail.ru|with non-English symbols |
+		|test@.      |invalid domain           |
+		|@mail.ru    |empty username           |
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_06 User did not check the "I agree to the personal data processing" checkbox
+	Given user filled all form with correct data
+	And user attached CV in correct format
+	When user did not check checkbox "I agree to the personal data processing"
+	And user pressed "Send" button
+	Then "I agree to the personal data processing" text near the checkbox should become red
+	And the form is not sent, e.g. "Thanks for being awesome!" message does not appear
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_07 Attach incorrect CV file
+	Given user filled all form with correct data
+	When user attached CV file with the following parameters: <description>
+	And user pressed "Send" button
+	Then a message should appear: "Something went wrong"
+	Examples:
+		|description                                  |
+		|file with .jpeg extension - wrong file format|
+		|file with .pdf extension, >10 Mb size        |
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_08 User did not attach CV file
+	Given user filled all form with correct data
+	When user did not attach CV file
+	And user pressed "Send" button
+	Then text on the "Attach CV" button should become red
+	And the form is not sent, e.g. "Thanks for being awesome!" message does not appear
+
+@Type:Negative
+@Priority:Critical
```

Comment:
I'm not sure that priority is critical.

Reply:
Yes, agree. At first I supposed it's the same as not attached CV